src/twisted/logger/_filter.py

```diff
@@ -154,15 +151,13 @@ def logLevelForNamespace(self, namespace):
         C{logLevelForNamespace("mypackage.subpackage")} will return
         C{LogLevel.debug}.
 
-        @param namespace: A logging namespace, or L{None} for the default
+        @param namespace: A logging namespace.  Use C{""} for the default
```

Comment:
Stop using `None` as the default namespace, as it's distinction from `""` is subtle and unintentional and I found a bug here that resulted from that distinction.